Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "Apache-2.0"
 readme = "README.md"
 version = "0.0.10"
 edition = "2021"
-rust-version = "1.67.0"
+rust-version = "1.81.0"
```

Comment:
Change in MSRV is required to use core::error package that was only stabilised in this version of Rust.